File: output/gemma4-e2b/avatar/Python/Java/codeforces_379_A.java
import java.util.Scanner;

public class codeforces_379_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Read candeleIntere and b from a single line of input
        if (scanner.hasNextInt()) {
            int candeleIntere = scanner.nextInt();
            if (scanner.hasNextInt()) {
                int b = scanner.nextInt();
                
                int s = 0;
                int restoSciolte = 0;
                
                while (candeleIntere > 0 || restoSciolte >= b) {
                    // candeleIntere += restoSciolte // b
                    candeleIntere += restoSciolte / b;
                    
                    // restoSciolte %= b
                    restoSciolte %= b;
                    
                    // s += candeleIntere
                    s += candeleIntere;
                    
                    // restoSciolte += candeleIntere % b
                    restoSciolte += candeleIntere % b;
                    
                    // candeleIntere //= b
                    candeleIntere /= b;
                }
                
                System.out.println(s);
            }
        }
        
        scanner.close();
    }
}

File: output/gemma4-e2b/avatar/Java/Python/codeforces_379_A.py
import sys

def solve():
    # Read a and b from standard input
    try:
        # Read the line and split it into two integers
        data = sys.stdin.read().split()
        if len(data) < 2:
            return
        a = int(data[0])
        b = int(data[1])
    except:
        # Handle potential reading errors if input format is unexpected
        return

    res = a
    
    # The loop condition and operations mirror the Java logic
    while a >= b:
        # res += (a / b)  (Integer division in Python is //)
        res += a // b
        
        # a = (a / b) + (a % b)
        a = (a // b) + (a % b)
        
    print(res)

solve()